src/templates/invoice/Invoice.tsx

```diff
@@ -0,0 +1,160 @@
+  font-family: "Lucida Sans Unicode", "Lucida Grande", sans-serif;
+  color: #4e4e50;
+  width: 100%;
+  height: 100vh;
+  max-height: 1000px;
+  max-width: 980px;
+  position: relative;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+`;
+
+export const InvoiceTemplate: FunctionComponent<TemplateProps<Invoice> & { className?: string }> = ({
+  document,
+  className = ""
+}) => {
+  const { id, date, customerId, terms, billFrom, billTo, invoiceItems } = document;
+
+  return (
+    <div>
+      <div css={container} className={className}>
+        <h1 style={{ color: "#4172AF", textAlign: "right" }}>INVOICE</h1>
+        <div className="m-0" style={{ position: "absolute", top: "0" }}>
+          <h1 style={{ color: "#4172AF", textAlign: "left" }}>{billFrom?.companyName}</h1>
+          <h3 style={{ textAlign: "left" }}>{billFrom?.streetAddress}</h3>
+          <h3 style={{ textAlign: "left" }}>
+            {billFrom?.city}
+            {`, `}
+            {billFrom?.postalCode}
+          </h3>
+          <h3 style={{ textAlign: "left" }}>{billFrom?.phoneNumber}</h3>
+        </div>
+        <table
+          className="table"
+          style={{ width: "40%", right: "0", top: "100px", position: "absolute", textAlign: "center" }}
+        >
+          <thead style={{ backgroundColor: "#4172AF", color: "white" }}>
+            <tr>
+              <th scope="col">INVOICE #</th>
+              <th scope="col">DATE</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>{id}</td>
+              {date && <td>{format(new Date(date), "d MMM yyyy")}</td>}
+            </tr>
+          </tbody>
+        </table>
+        <table
+          className="table"
+          style={{ width: "40%", right: "0", top: "180px", position: "absolute", textAlign: "center" }}
+        >
+          <thead style={{ backgroundColor: "#4172AF", color: "white" }}>
+            <tr>
+              <th scope="col">CUSTOMER ID</th>
+              <th scope="col">TERMS</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>{customerId}</td>
+              <td>{terms}</td>
+            </tr>
+          </tbody>
+        </table>
+        <table className="table" style={{ width: "40%", left: "0", top: "120px", position: "relative" }}>
+          <thead style={{ backgroundColor: "#4172AF", color: "white" }}>
+            <tr>
+              <th scope="col">BILL TO</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>{billTo?.name}</td>
+            </tr>
+            <tr>
+              <td>{billTo?.company.companyName}</td>
+            </tr>
+            <tr>
+              <td>{billTo?.company.streetAddress}</td>
+            </tr>
+            <tr>
+              <td>
+                {billTo?.company.city}
+                {`, `}
+                {billTo?.company.postalCode}
```

Comment:
try to group the comma with last showable item? so it won't render just a comma if `billTo.company.city` and billTo.company.postalCode` are not present